bauro-ansible/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
If you add something like `--extra_vars "names_file=$1"` you don't need the variable in host_vars.